src/reference/asciidoc/amqp.adoc

```diff
@@ -1149,7 +1149,14 @@ To configure a default user id for outbound messages, configure it on a `RabbitT
 Similarly, to set the user id property on replies, inject an appropriately configured template into the inbound gateway.
 See the http://docs.spring.io/spring-amqp/reference/html/_reference.html#template-user-id[Spring AMQP documentation] for more information.
 
-
+[[amqp-delay]]
+=== Delayed Message Exchange
+
+Spring AMQP supports the http://docs.spring.io/spring-amqp/docs/1.6.3.RELEASE/reference/html/_reference.html#delayed-message-exchange[RabbitMQ Delayed Message Exchange Plugin].
```

Comment:
I find this link as much better: http://docs.spring.io/spring-amqp/reference/html/_reference.html#delayed-message-exchange.
At least it is version-free.